src/tests/components/ProjectForm.test.js

```diff
@@ -21,4 +23,50 @@ describe('ProjectForm', () => {
     form.simulate('submit')
     expect(props.handleSubmit).toHaveBeenCalledTimes(1)
   })
+
+  it('adds new repo fields', () => {
+    const wrapper = mount(
+      <Provider store={store}>
+        <ConnectedProjectForm />
+      </Provider>
+    )
+    wrapper.find('.ui.button.field-group__add').first().simulate('click')
+    expect(wrapper.find('Field[name="repos[0].value"]').length).toBeTruthy()
+    wrapper.find('.ui.button.field-group__add').first().simulate('click')
+    expect(wrapper.find('Field[name="repos[1].value"]').length).toBeTruthy()
```

Comment:
can we avoid using multiple `expects` in the same it block?
https://softwareengineering.stackexchange.com/questions/7823/is-it-ok-to-have-multiple-asserts-in-a-single-unit-test